package.json

```diff
@@ -26,9 +29,9 @@
   "scripts": {
     "test": "make test"
   },
-  "repository": "git://github.com/baugarten/node-restful",
+  "repository": "git://github.com/maxaille/node-restful",
```

Comment:
It send back the new updated object instead of the untouched one. It's a modification from mongoose 3 to 4. See http://mongoosejs.com/docs/migration.html